src/XliffReplacer/SdlXliffSAXTranslationReplacer.php

```diff
@@ -161,7 +161,8 @@ protected function prepareTranslation( $seg, $transUnitTranslation = "" ) {
             $translation = $segment;
         } else {
             if ( $this->callback ) {
```

Comment:
If this Callback is a class, and not a function ( but it's name is callback ), this `if` is too weak, please check for an instance of the right class/interface and change the name to be consistent.

Reply:
Done